Sources/Extensions/Shared/ColorExtensions.swift

```diff
@@ -92,7 +92,7 @@ public extension Color {
 	/// SwifterSwift: Short hexadecimal value string (read-only, if applicable).
 	public var shortHexString: String? {
 		let string = hexString.replacingOccurrences(of: "#", with: "")
-		let chrs = Array(string.characters)
+		let chrs = Array(string)
```

Comment:
Won't this fail now?

Reply:
I think not, it returns an array of characters the same way as before :)